framework/apis/config/v1alpha1/config.proto

```diff
@@ -30,6 +30,8 @@ message Limiter {
   }
   RateLimitBackend backend = 3;
   google.protobuf.Duration refresh = 4 [(gogoproto.stdduration) = true];
+  bool DisableAdaptiveRateLimit = 5;
+  bool DisableGlobalRateLimit = 6;
```

Comment:
@cywang1905 we'd better move module api to its own repo soon

Reply:
sure, it's not graceful to change framework when module config changes